runtime/src/lib.rs

```diff
@@ -26,6 +26,7 @@ use std::{
 use thiserror::Error;
 
 pub mod deterministic;
+pub mod metrics;
```

Comment:
moved this to `runtime` from `utils` since I needed to use the `Clock` trait for histogram, but idk it might just be a lost cause

Reply:
Okay I'm not sure if it's better in `commonware-utils` or `commonware-runtime` since `runtime` already imported `prometheus` but `utils` didn't...

Reply:
Let's keep it here and call it `telemetry` instead of `metrics`. Given we are treating telemetry as a first-rate concern (and have already enshrined it into runtime), this is the best place (to avoid all sorts of circular dependencies).